Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -41,26 +41,39 @@ public extension UIConfig {
 
 public extension UIConfig {
     struct ColorPalette {
+        // MARK: - General
+
+        public var subtitleText: UIColor = .lightGray
+        public var text: UIColor = .black
+        public var generalBackground: UIColor = UIColor(rgb: 0xfcfcfc)
+        public var shadow: UIColor = .lightGray
```

Comment:
Please refer to `streamXXX` color here instead.

Reply:
Thanks, done.